### DISCUSSION TITLE
Sync media deletion across devices

### DISCUSSION THREAD

**Discussion by danielfernau** [ente-io/ente#3425] [Enhancements]
It would be nice to have the mobile app suggest deleting media from local storage that has been moved to the trash via the desktop app or another mobile device logged in to the same account. This option could also be added to the "Free up space" section of the app.

**Comment by ua741** [comment]
Tracking here: https://github.com/ente-io/ente/discussions/3242

**Reply by danielfernau** [reply]
Thanks for the follow-up - but my feature request above is in no way related to #3242, as I'm not referring to the issue described in that ticket. My suggestion was to have a feature where one can have a button in the app that when clicked offers to delete the local versions of photos from that device which were moved to the trash on another device.

Scenario: I delete 123 photos via the desktop app. Afterwards, the Android/iOS app offers something like "123 photos have been deleted from the library. Would you like to delete the local copies from this device as well?"

It's intended to streamline the library and storage management process. Not related to any issue with (re)syncing already deleted stuff.

**Reply by ua741** [reply]
@danielfernau 
Ack. Sorry, I closed this issue because during internal discussion, we figure out it make sense to implement all these related features together. I will reopen this request again.

Your request similar to https://github.com/ente-io/ente/discussions/900

**Reply by danielfernau** [reply]
No worries, thanks! Yes, technically these can be bundled as it's the same area code-wise, just different features. And also yes, basically the same as #900. Feel free to reach out any time if you need further input! 👍
